drivers/SmartThings/zwave-siren/src/ecolink-wireless-siren/init.lua

```diff
@@ -68,6 +68,23 @@ local do_refresh = function(self, device)
   end
 end
 
+local function siren_on(self, device, cmd)
+  device:send_to_component(Basic:Set({value = 0xFF}), cmd.component)
+  device:send_to_component(SwitchBinary:Get({}), cmd.component)
+  local endpoint = component_to_endpoint(device, cmd.component)
+  if endpoint[1] == 2 then
```

Comment:
I think it would look better to switch this on `cmd.component` (`"siren1"` I guess?) rather than the endpoint number